tests/feature/test_tags.py

```diff
@@ -127,7 +127,7 @@ def i_have_bar():
         scenarios('test.feature')
     """)
     result = testdir.runpytest('-rsx')
-    result.stdout.fnmatch_lines(["SKIP *: Not implemented yet"])
+    result.stdout.fnmatch_lines(["SKIP*: Not implemented yet"])
```

Comment:
On `pytest==4.1.1` message in `short test summary info` is:
```
SKIP [1] test_apply_tag_hook.py:40: Not implemented yet
```

But on `pytest==4.2.0` it is:
```
SKIPPED [1] test_apply_tag_hook.py:40: Not implemented yet
```

I fixed above test to work with both messages.